### PR TITLE
Lazy load update

### PR DESCRIPTION
**Pull request by DomSheehy**

### CODE REVIEW

File: dist/owl.carousel.js [hunk 1]
```diff
@@ -1897,6 +1897,8 @@
 		$elements.each($.proxy(function(index, element) {
 			var $element = $(element), image,
 				url = (window.devicePixelRatio > 1 && $element.attr('data-src-retina')) || $element.attr('data-src');
+				sizes = $element.attr('data-sizes'),
+				srcset = $element.attr('data-srcset');
```
Comment (lebobbi) [inline dist/owl.carousel.js]: Are sizes and srcset global? If not, maybe the ; after the url definition was missed

File: src/js/owl.lazyload.js [hunk 2]
```diff
@@ -91,6 +91,8 @@
 		$elements.each($.proxy(function(index, element) {
 			var $element = $(element), image,
 				url = (window.devicePixelRatio > 1 && $element.attr('data-src-retina')) || $element.attr('data-src');
+				sizes = $element.attr('data-sizes'),
+				srcset = $element.attr('data-srcset');
```
Comment (lebobbi) [inline src/js/owl.lazyload.js]: same as above

Comment (DomSheehy) [reply src/js/owl.lazyload.js]: they're defined/used on a per element basis

Comment (DomSheehy) [reply src/js/owl.lazyload.js]: Ah youre right, i was testing locally!!!

**Pull request merged**